odoo/netsvc.py

```diff
@@ -129,6 +129,9 @@ def record_factory(*args, **kwargs):
 
     logging.addLevelName(25, "INFO")
     logging.captureWarnings(True)
+    # enable deprecation warnings but ignore invalid escapes (there's
+    # a ton of them and that's a really low-value signal, probably)
+    warnings.filterwarnings('once', r'^(?!invalid escape sequence \\.)', category=DeprecationWarning)
```

Comment:
+1, most of those warnings come from single `\` in docstrings and unnecessary raw-string in regexps. Some editors use a different color-schema for raw-strings to highlight regexps expressions, this is very convenient and shouldn't be treated as an error.

Reply:
the opposite, in a rawstring `\` doesn't create an escape. So this generally comes from *missing* rawstrings e.g. https://github.com/odoo/odoo/blob/fccaa703c4927ca906ccef329725ab00d3262051/addons/hr_expense/models/hr_expense.py#L565
since the string is not raw, python parses `\d` as an escape, but since it's not a valid escape it triggers a DeprecationWarning then leaves the entire thing as-is. This should either be a rawstring, or the backslash should itself be escaped. But fixing it would require a lot of editions and would not be very useful, I've not heard that there's an intention to make such invalid escapes an error by default as it is in other languages so it doesn't matter much.